.env.development

```diff
@@ -8,4 +8,6 @@ ACCESS_TOKEN_COOKIE_NAME='edx-jwt-cookie-header-payload'
 USER_INFO_COOKIE_NAME='edx-user-info'
 SEGMENT_KEY=''
 DESIGNER_BASE_URL='http://designer.app:18808'
-UNBRANDED_LANDING_PAGE=false
+HOSTNAME='example.com'
```

Comment:
I forgot that people were not using Docker when removing this and adding it to the compose.  Sorry if this caused issues, I will update the Readme later to add that you have to set HOSTNAME in the compose if you are using Docker.